### ISSUE TITLE
fio fails to build when using Ubuntu 16.04

### ISSUE THREAD

**Issue by meteorfox** [GoogleCloudPlatform/PerfKitBenchmarker#1094]
I'm using a recent version of the Ubuntu 16.04 LTS image on OpenStack, and FIO is failing to build with the following error:

```
Makefile:287: recipe for target 'lib/libmtd_legacy.o' failed
STDERR: Warning: Permanently added '10.50.50.55' (ECDSA) to the list of known hosts.
FIO_VERSION = fio-2.2.10
In file included from lib/libmtd_legacy.c:38:0:
lib/libmtd.h:288:8: error: unknown type name 'uint8_t'
        uint8_t mode);
        ^
lib/libmtd.h:305:4: error: unknown type name 'uint64_t'
    uint64_t start, uint64_t length, void *data);
    ^
lib/libmtd.h:305:20: error: unknown type name 'uint64_t'
    uint64_t start, uint64_t length, void *data);
                    ^
lib/libmtd.h:322:5: error: unknown type name 'uint64_t'
     uint64_t start, uint64_t length, void *data);
     ^
lib/libmtd.h:322:21: error: unknown type name 'uint64_t'
     uint64_t start, uint64_t length, void *data);
                     ^
make: *** [lib/libmtd_legacy.o] Error 1
```
After I quick Google of the error, this came up
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=815735

Apparently, the fix is already in fio (`#include <stdint.h>`)

## Steps to recreate

1. Add official Ubuntu Xenial 16.04 image (http://docs.openstack.org/image-guide/obtain-images.html) as `ubuntu-1604-lts`
2. Run `fio` benchmark against, in this case, OpenStack provider
```bash
python pkb.py --cloud=OpenStack --machine_type=m1.medium --image='ubuntu-1604-lts'\
              --openstack_network=private --openstack_floating_ip_pool=public \
              --benchmarks=fio
```

**Comment by meteorfox** [comment]
According to the bug report linked above, this issue was fixed iin version fio/2.6-1

**Comment by meteorfox** [comment]
The tag `fio-2.7` is the one that contains the fix.

See https://github.com/axboe/fio/commit/d7bb6180f831091c468e5aa749b142efd5eddda4 for more info
